synapse/lib/stormtypes.py

```diff
@@ -1604,8 +1604,12 @@ async def _min(self, *args):
                 continue
             vals.append(arg)
 
+        if len(vals) < 1:
+            mesg = '$lib.min() must have at least one argument or a list containing at least one value.'
+            raise s_exc.StormRuntimeError(mesg=mesg)
```

Comment:
Seems like `s_exc.BadArg` would be a better exception type here (and in `max()` below) since the problem is a bad argument, not an issue with the runtime.

Reply:
We don't know we're in a bad shape until we've processed all the args; so its not like one particular arg in a args list was the bad one. I could go either way here.

Reply:
StormRuntimeError seems reasonable to me since it isn't necessarily any one of the args in particular